rust/src/api.rs

```diff
@@ -500,6 +501,37 @@ impl<'a> Endpoint<'a> {
         .await?;
         Ok(())
     }
+
+    pub async fn transformation_get(
+        &self,
+        app_id: String,
+        endpoint_id: String,
+    ) -> Result<EndpointTransformationOut> {
+        Ok(endpoint_api::get_endpoint_transformation_api_v1_app_app_id_endpoint_endpoint_id_transformation_get(
+			self.cfg,
+			endpoint_api::GetEndpointTransformationApiV1AppAppIdEndpointEndpointIdTransformationGetParams {
+				app_id,
+				endpoint_id,
+				idempotency_key: None,
+			}
+		)
+		.await?)
+    }
```

Comment:
Nit: formatting looks a bit strange here, some lines used tabs it seems?

Reply:
Weird `cargo fmt` didn't catch that. Fixed :)